graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetConfig.java

```diff
@@ -34,11 +34,10 @@
 import javax.validation.constraints.NotNull;
 import java.time.ZonedDateTime;
 
-import static java.util.Objects.requireNonNull;
-
 @AutoValue
 @WithBeanGetter
 @JsonAutoDetect
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Otherwise loading the `IndexSetConfig` from the database failed due to the unknown `default` field.